doc/classes/BaseMaterial3D.xml

```diff
@@ -268,6 +274,7 @@
 		</member>
 		<member name="metallic_texture" type="Texture2D" setter="set_texture" getter="get_texture">
 			Texture used to specify metallic for an object. This is multiplied by [member metallic].
+			[b]Note:[/b] When assigned in the editor, other textures' properties will automatically be set based on their texture name. For example, if you assign [code]res://levels/wood_albedo.png[/code] to [member albedo_texture], then [code]res://levels/wood_normal.png[/code] will be assigned to [member normal_texture] and so on. This is based on common texture name conventions, so most texture websites should be handled automatically out of the box.
```

Comment:
Can you modify the example for the specific cases? Is this metallic_texture or albedo texture?